theme/SubsiteRow.scss

```
@@ -0,0 +1,78 @@
+// Visual treatment for the SubsiteRow icons.
+//
```

Stylelint (17.12.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)